src/version.c

```diff
@@ -34,36 +34,43 @@
 
 #define MAX_VERSION_STR_SIZE 11
 
-int get_server_format(int server_version)
+int get_int_from_url(const char *url)
```

Comment:
since this is a public function it would be better to follow the standard we adopted of starting the function name with something common for all functions of this file ("version").

For example maybe something like: `version_from_url`, I think the "int" part is implied by the return type so no need to reference it.

Reply:
I don't like the name version_from_url() because that's not what this function does. This function is really reading an int from an url. It doesn't matter if it's a version or not. The ideal would be to reorganize several things in version.c to improve that, but I don't think it's worth to do it right now. I would leave it like this for now.